### DISCUSSION TITLE
Grain growth in neper

### DISCUSSION THREAD

**Discussion by kumaranvenk** [neperfepx/neper#837] [General]
IN order to show grain growth, suggested code is **"diameq:lognormal(1,0.35)"**, what does 1 and 0.35 represents. Also, if i have an average growth of 2 micron in grain, how to write code for that??

**Comment by rquey** [comment]
1 is the mean and 0.35 the standard deviation. For 2 microns, and with units in microns, it would be `"diameq:lognormal(2,0.70)"` and use `-n from_morpho` so as tp set the number of grains from the grain size distribution. For a cube of 10 x 10 x 10 microns:
```
neper -T -n from_morpho -morpho "diameq:lognormal(2,0.70),1-sphericity:lognormal(0.145,0.03)" -domain "cube(10,10,10)"
```
![d](https://github.com/neperfepx/neper/assets/2659376/d32ea55d-d53f-4432-acbe-1b8d45da5acb)
